xarray/core/common.py

```diff
@@ -1423,17 +1440,30 @@ def astype(self, dtype, casting="unsafe", copy=True, keep_attrs=True):
         out : same as object
             New object with data cast to the specified type.
 
+        Notes
+        -----
+        The ``order``, ``casting``, ``subok`` and ``copy`` arguments are only passed
```

Comment:
@pydata/xarray are we ok with this as a general policy for functions that are light wrappers of the underlying array-type's method.

Reply:
Hmm true we don't really do this in other places AFAIK. However, it is similar to passing `**kwargs` on.

Reply:
This looks like a good solution to me!